app/services/processors/completed_review_turnaround_pull_request_backfiller.rb

```diff
@@ -0,0 +1,9 @@
+module Processors
```

Comment:
minor comment but this code (code used in rake tasks in general) can go in `lib/tasks` next to the rake task because it doesn't need to be autoloaded with the rest of the app since it's only going to be used by the rake task